circle.yml

```diff
@@ -61,6 +61,13 @@ test:
         parallel: true
     - codecov:
         parallel: true
-    - rm -rf docs/API && make -C docs clean
+    - pyenv local 3.4.3:  # sphinx fails with 3.5.1
```

Comment:
This (re)installs Python 3.4.3 on both nodes (it is already installed on one of them), which means Sphinx is run on the same Python version on two nodes. That gives us nothing. Just dont run it on one node.